firebase.json

```
@@ -5,17 +5,20 @@
   },
   "functions": {
     "source": "dist/functions",
-    "predeploy": ["npm run check", "npm run build"]
+    "predeploy": [
+      "npm i",
```

#53 にコメントしたものと同じ理由です :bow:

```suggestion
      "npm ci",
```

---

:+1: